EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs

```diff
@@ -31,12 +31,23 @@ public class InteractingScp330EventArgs : IPlayerEvent, IScp330Event, IDeniableE
         public InteractingScp330EventArgs(Player player, int usage)
         {
             Player = player;
-            Scp330 = Scp330Bag.TryGetBag(player.ReferenceHub, out Scp330Bag scp330Bag) ? (Scp330)Item.Get(scp330Bag) : null;
-            Candy = Scp330Candies.GetRandom();
             UsageCount = usage;
             ShouldSever = usage >= 2;
             ShouldPlaySound = true;
             IsAllowed = Player.IsHuman;
+
+            if (Scp330Bag.TryGetBag(player.ReferenceHub, out Scp330Bag scp330Bag))
+            {
+                Scp330 = (Scp330)Item.Get(scp330Bag);
+            }
```

Comment:
```suggestion
                Scp330 = (Scp330)Item.Get(scp330Bag);

```

Reply:
Stylecop doesn't like this

Reply:
> Stylecop doesn't like this

And I don't like stylecop.

Reply:
☠️☠️☠️💀

Reply:
> > Stylecop doesn't like this
> 
> And I don't like stylecop.

He can't be cause he have multiple line in the else